enos/modules/iam_setup/main.tf

```diff
@@ -0,0 +1,43 @@
+data "aws_caller_identity" "current" {}
+
+variable "test_id" {}
+
+locals {
+  user_email = split(":", data.aws_caller_identity.current.user_id)[1]
```

Comment:
Note: Last outstanding thing with Enos - working with Josh to figure out how to make this work in CI